cmd/ocsp-updater/main.go

```diff
@@ -501,7 +503,8 @@ type looper struct {
 
 func (l *looper) tick() {
 	tickStart := l.clk.Now()
-	err := l.tickFunc(l.batchSize)
+	ctx := context.TODO()
```

Comment:
It may be useful to have a maximum time for a tick, so I wasn't quite sure that Background was appropriate.